### PR TITLE
Adding missing .background-icon-base mixin;

### LINKED ISSUE #413: Notice: close button icon is missing
Version affected: 6.x

Very quick investigation suggests that `.background-icon-base;` might be missing. If so this fix could go out in next patch release.

Screenshot from current website:

screen shot 2018-10-15 at 1 03 49 pm

Screenshot from archive (5.x):

screen shot 2018-10-15 at 1 05 08 pm

### PR DESCRIPTION
**Pull request by seangates**
## Description
- adds back the missing close icon in the flyout notice, which was accidentally removed

## References
Fixes #413.

## Screenshots

**Broken**

screen shot 2018-10-15 at 1 03 49 pm

**Fixed**

image

### CODE REVIEW

**Comment by ianmcburnie** [comment]
@seangates The close icon looks a bit lower than in version 5.x (you can compare with the screenshot in my issue)

**Comment by seangates** [comment]
@ianmcburnie Updated. Please check once more.

File: docs/static/ds6/docs.js [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
-$_mod.installed("@ebay/skin$6.0.4","makeup-expander","0.4.0");
-$_mod.main("/makeup-expander$0.4.0","");
-$_mod.installed("makeup-expander$0.4.0","custom-event-polyfill","0.3.0");
+$_mod.installed("@ebay/skin$6.0.4","makeup-expander","0.5.0");
```
Comment (ianmcburnie) [inline docs/static/ds6/docs.js]: Let's remove this generated docs.js file from the PR, as master branch is still on version 0.4.0 of expander (I only updated it in 6.1.0 branch).

**Comment by seangates** [comment]
@ianmcburnie Reverted. Please check once more. 👍

**Pull request merged**